portality/formcontext/forms.py

```diff
@@ -373,13 +385,13 @@ class PublicApplicationForm(JournalInformation, Suggestion, PublicSuggester):
     pass
 
 
-class ManEdApplicationReviewForm(Editorial, Workflow, ApplicationOwner, JournalInformation, Suggestion, Subject, AdminSuggester, Notes):
+class ManEdApplicationReviewForm(Editorial, Workflow, ApplicationOwner, JournalInfoOptionalPaymentURLs, Suggestion, Subject, AdminSuggester, Notes):
     """
     Managing Editor's Application Review form.  It consists of:
         * Editorial - ability to add editorial groups (but ability to add editors individually will be disabled)
         * Workflow - ability to change application status
         * ApplicationOwner - able to set the owner, and for the owner assignment to be optional except under certain circumstances
-        * JournalInformation - journal bibliographic data
+        * ManEdJournalInfoOverride - journal bibliographic data, with custom validation for managing editors
```

Comment:
Yes good point.  Forgot about the documentation. If it's easy to fix go
ahead otherwise the code is the key bit.

Cheers, Richard

Richard Jones

Founder, Cottage Labs
t: @cottagelabs
w: http://cottagelabs.com
On 20 Jan 2015 17:11, "Emanuil Tolev" <notifications@github.com> wrote:

> In portality/formcontext/forms.py
> <https://github.com/DOAJ/doaj/pull/530#discussion-diff-23237462>:
>
> >      """
> >      Managing Editor's Application Review form.  It consists of:
> >          * Editorial - ability to add editorial groups (but ability to add editors individually will be disabled)
> >          * Workflow - ability to change application status
> >          * ApplicationOwner - able to set the owner, and for the owner assignment to be optional except under certain circumstances
> > -        * JournalInformation - journal bibliographic data
> > +        * ManEdJournalInfoOverride - journal bibliographic data, with custom validation for managing editors
>
> the name's changed to JournalInfoOptionalPaymentURLs, not
> ManEdJournalInfoOverride. I can fix it in the hotfix branch if you like.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DOAJ/doaj/pull/530/files#r23237462>.
>